force-app/main/default/classes/DummyJSONCallout.cls

```diff
@@ -93,23 +150,50 @@ public with sharing class DummyJSONCallout {
      */
 	
 	public static void postCreateDummyJSONUser(String contactId) {
-		// Create HTTP request to send.
 
-		// Set the endpoint URL. Use direct URL or for best practices use Named Credential.
+		Contact cont = [SELECT Id, FirstName, LastName, Email, Phone FROM Contact WHERE Id = :contactId LIMIT 1];
 
+		// Create HTTP request to send.
+		HttpRequest request = new HttpRequest();
+		// Set the endpoint URL. Use direct URL or for best practices use Named Credential.
+		request.setEndpoint('https://dummyjson.com/users/add');
 		// Set the HTTP method to POST.
+		request.setMethod('POST');				
+		request.setHeader('Content-Type', 'application/json');
+		// Set the body using generateDummyJsonUserPayload method.					
+		request.setBody('{"firstName": "Tester"}');
```

Comment:
This is not setting the correct body. This is always the same value, not the values from the contact!